### PR TITLE
refactor: revert to plain emotion

### PR DESCRIPTION
**Pull request by suevalov**
# Purpose of PR

It turned out that using `@emotion/core` makes it impossible to override styles with simple `emotion` which makes it a big deal and a breaking change for our internal Contentful applications.

Also, @denkristoffer found an issue https://github.com/emotion-js/emotion/issues/2275 that tells that `css=` property is causing performance regressions.

### CODE REVIEW

File: packages/components/badge/src/getBadgeStyles.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import tokens from '@contentful/f36-tokens';
-import { Interpolation } from '@emotion/core';
+import type { Interpolation } from '@emotion/serialize';
```
Comment (denkristoffer) [inline packages/components/badge/src/getBadgeStyles.ts:2]: Should this be added as an explicit dependency?

**Review by mshaaban0 — APPROVED** [review]

File: packages/components/badge/src/getBadgeStyles.ts (hunk 1 above)
Comment (suevalov) [reply packages/components/badge/src/getBadgeStyles.ts]: good point, I decided to remove explicit type here

**Review by denkristoffer — APPROVED** [review]
🙏

File: scripts/plop-templates/package/package.json.hbs [hunk 2]
```diff
@@ -23,7 +23,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.6.2",
-    "@emotion/core": "^10.1.1",
+    "emotion": "^10.0.17",
```
Comment (denkristoffer) [inline scripts/plop-templates/package/package.json.hbs:26]: A pedantic nit here, if we don't sort this alphabetically yarn will do it for us and create a diff on the first run, which adds noise.

Comment (suevalov) [reply scripts/plop-templates/package/package.json.hbs]: updated 🧐

**Pull request merged**